eogrow/pipelines/split_grid.py

```diff
@@ -59,7 +61,14 @@ class Schema(Pipeline.Schema):
                 " not all area manager classes are supported."
             ),
         )
-        # TODO: could add AOI file?
+
+        keep_aoi_only: bool = Field(
```

Comment:
Just to understand better: in https://github.com/sentinel-hub/eo-grow/pull/145#discussion_r1038185372 we were discussing if the user could provide an AOI for some further filtration - I thought you meant this for any generic AOI, meaning that the user could also provide a sub-part of the original AOI, but IIRC now the user only has the option to keep or discard eopatches which would fall out of the AM-AOI after splitting?

Reply:
yes, we were discussing many options. Technically the manager-based filtration is always desired. The custom AOI filtration not that often. Since we're outputting a gpkg, the user can always modify it to a sub-area by hand if desired (some redundant data needs to be deleted as well)

Or did you get too excited about that, and now I need to add it? :P